core/lib.deno_core.d.ts

```diff
@@ -21,6 +21,18 @@ declare namespace Deno {
       b?: any,
     ): Promise<any>;
 
+    /** Mark following promises as "ref", ie. event loop won't exit
+     * until they are resolved. All async ops are "ref" by default. */
+    function refOps(
+      ...promiseIds: number[]
```

Comment:
Why does it take multiple `promiseId`s?
I do understand that it is faster to coalesce multiple `refOp` calls into a single call, but I wonder how common that is, and also note that passing a single int into the binding layer is faster than passing an array.

Reply:
In some resources we might have multiple pending ops associated with it. So instead of crossing boundary multiple times I thought it's better to do it in one go. Keep in mind that this is spread operator so you're passing multiple integers and not an array (`Deno.core.refOps(1, 2, 3, 4)`). Also is there a situation where this API would be used in a hot path?

Reply:
> Keep in mind that this is spread operator so you're passing multiple integers and not an array (Deno.core.refOps(1, 2, 3, 4))

I see, in the binding layer you're just iterating over `FunctionCallbackArguments`, that's pretty efficient and I have no concerns about it.

> In some resources we might have multiple pending ops associated with it.

You'll find that this almost never happens in practice but oh well.

Reply:
If you prefer to have it accept single ID that is fine for me as well. Your call